src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -96,8 +103,12 @@ public Controls(Subsystems s) {
     }
 
     public void bindClimbControls() {
-        climbFixedArmDown.whenPressed(new RetractArmFullyCommand(subsystems.climbSubsystem));
-        climbFixedArmUp.whenPressed(new ExtendArmCommand(subsystems.climbSubsystem));
+        climbArmDown.whenPressed(new RetractArmFullyCommand(subsystems.climbSubsystem));
+        climbArmUp.whenPressed(new ExtendArmCommand(subsystems.climbSubsystem));
+
+        climbArmDownManual.whileHeld(new ClimbSetArmCommand(subsystems.climbSubsystem, -0.4));
+        climbArmUpManual.whileHeld(new ClimbSetArmCommand(subsystems.climbSubsystem, 0.4));
```

Comment:
These tell the climb arm to go to position -0.4 or position 0.4, which seems incorrect. ClimbSetArmCommand currently sets an absolute position. I'm guessing there's missing code in ClimbSetArmCommand and ClimbSubsystem to enable this to instead simply set the motor power (or speed).